### ISSUE TITLE
Import settings not working

### ISSUE THREAD

**Issue by schweinharlm** [YePpHa/YouTubeCenter#1400]
I had to update Chrome (ver 39.0.2171.99) & when I went to re-add my old settings, the file will not work. It won't save my imported file. Using dev ver 470, chrome extension.

**Comment by KDsMax** [comment]
Make sure you import settings while on YouTube homepage, not any other page because it won't work.

**Comment by schweinharlm** [comment]
Updated to new ver of Chrome & made sure I was importing on YouTube homepage & still the same issue. Once I hit "save & reload" the page does not refresh/doesn't save import.

**Comment by Haysooooom** [comment]
Having the same issue here. I even deleted the old settings and exported a new one and still wouldn't get loaded.

**Comment by alpe12** [comment]
Same here...

**Comment by HowellBP** [comment]
Same issue in the latest developer version. Tried importing settings both via file and copy+paste, from https://www.youtube.com and other locations. Nothing worked.

**Comment by AwesomeDonkey** [comment]
Having this issue with multiple browsers (Chrome, Firefox, Safari, etc.) on Windows, Mac and Ubuntu.

**Comment by Sophira** [comment]
I'm also having this issue. I exported a settings file from:

* Firefox 22.0
* Greasemonkey 1.5
* YouTube Center Developer Build 416 (userscript version)

and am trying to import it into:

* Firefox 33.0.3
* Greasemonkey 2.3
* YouTube Center Developer Build 470 (userscript version)

When I point it at the exported .ytcs file from the previous setup, the "Save & Reload" button was greyed out but no attempt was made to load the settings. There is no output in the Browser Console when I click the button, even with all options (Errors, Warnings, and Log) turned on.

When loading the homepage, before trying to open the settings, these messages appear in the Browser Console:

```
window.controllers is deprecated. Do not use it for UA detection. https-everywhere.js:342
No such local preference: trigger_on_location_change
No such local preference: activations
No such local preference: body
No such local preference: version
ImageDocument is not defined Global_Allow_ImageDocuments_to_be_styled.user.js:9
Using //@ to indicate sourceMappingURL pragmas is deprecated. Use //# instead jquery-1.9.1.min.js:1
Use of getPreventDefault() is deprecated.  Use defaultPrevented instead. flashblock.xml:120
Use of Mutation Events is deprecated. Use MutationObserver instead. jquery-1.9.1.min.js:3
```

No further messages appear when I click the YouTube Center settings button and go through the procedure for importing my previous settings.

**Comment by Sophira** [comment]
It looks like I didn't have the "Logging" option enabled when I was looking at the console. In addition, I am now testing on a blank profile (so no other addons can affect it) and have updated to build 471. Now that I've turned on logging, there are a lot more messages when I load the YouTube homepage:

```
"Domain registered www.youtube.com." YouTubeCenter.js:26208
"[Property Wrapper] Wrapping the function undefined into a function wrapper." www.youtube.com:2021
"[Property Wrapper] Property doesn't exists in parent, attaching trigger using `getter and setter`." www.youtube.com:2021
"[PageReadinessListener] At event => headerInitialized" "feed_what_to_watch" www.youtube.com:2021
"Language set to en-US because it could not be auto-detected yet" www.youtube.com:2021
"[PageReadinessListener] At event => bodyInitialized" "feed_what_to_watch" www.youtube.com:2021
"Adding player shortcuts to document" www.youtube.com:2021
"[External] Setting player configruation." www.youtube.com:2021
"[SPF] ready" null www.youtube.com:2021
"[Property Wrapper] Property doesn't exists in parent, attaching trigger using `getter and setter`." www.youtube.com:2021
"[PageReadinessListener] At event => bodyInteractive" "feed_what_to_watch" www.youtube.com:2021
"[Thumbnail] Setup has begun..." www.youtube.com:2021
"[Storage] Checking if settings have expired." www.youtube.com:2021
"Checking for updates in -0.006321944444444445 hours..." www.youtube.com:2021
"Checking for updates now..." www.youtube.com:2021
"[Update] This is a dev build." www.youtube.com:2021
"Checking for updates..." www.youtube.com:2021
"[Title Listener] "" => "YouTube"" www.youtube.com:2021
"[Storage] Saving Settings" www.youtube.com:2021
"[Tab Events] Sending new settings to other open tabs." www.youtube.com:2021
"[Storage] Checking if settings have expired." www.youtube.com:2021
An unbalanced tree was written using document.write() causing data from the network to be reparsed. For more information https://developer.mozilla.org/en/Optimizing_Your_Pages_for_Speculative_Parsing _default;sz=970x250;tile=1;ssl=1;dc_yt=1;kbsg=HPGB150201;kga=-1;kgg=-1;klg=en;kmyd=ad_creative_1;ytexp=931347,954200,905656,955100,945084,9406208,9405768;ord=5254621907588162:280
can't access dead object tab-firefox.js:104
"[Storage] Saving Settings" www.youtube.com:2021
"Got Update Response" www.youtube.com:2021
"[Update] Current dev build #471. Your build number #471" www.youtube.com:2021
"No new updates available" www.youtube.com:2021
"[PageReadinessListener] At event => bodyComplete" "feed_what_to_watch" www.youtube.com:2021
"[Guide] Configurating the state updater!" www.youtube.com:2021
"[Player Update] Checking if player exist!" www.youtube.com:2021
"[PageReadinessListener] Stopping interval" www.youtube.com:2021
"[External] Setting player configruation." www.youtube.com:2021
can't access dead object tab-firefox.js:104
"[Player Listener] Has begun the setup..." www.youtube.com:2021
"[onYouTubePlayerReady] => updateConfig" www.youtube.com:2021
"[Config Update] Updating as page feed_what_to_watch" www.youtube.com:2021
```

When I attempt to import the settings, I see these messages:

```
"[Tab Events] Sending new settings to other open tabs." www.youtube.com:2021
"[Storage] Checking if settings have expired." www.youtube.com:2021
```

The first line occurs shortly after I open the settings, and the second occurs when I click the "Save & Reload" button. I have verified that the new settings do not get saved (if they are ever even loaded) because reloading the page and watching a video shows that the settings are not in place, and the settings dialog confirms it.

I hope this additional information is useful!

**Comment by alpe12** [comment]
This helped me:
https://github.com/YePpHa/YouTubeCenter/issues/1408#issuecomment-72264384

**Comment by Sophira** [comment]
Thank you, alpe12! That worked beautifully.

I've posted details in that other issue.

**Comment by AwesomeDonkey** [comment]
This is still happening in 473. YePpHa, is this one easy to fix?

**Comment by Sophira** [comment]
I can probably contribute a pull request for this. Hang on a second.

**Comment by YePpHa** [comment]
This should now be fixed, see #1408.
